lesson08/assignment/my-yoga-app/src/components/PictureDetail.vue

```diff
@@ -0,0 +1,24 @@
+<script>
+export default {
+  name: 'picture',
+  props: ['picture'],
+
```

Comment:
For this, we want you to create a picture/image component with props and validation. So you'll want to specify each picture prop, and use prop validation to verify the type. You are very close on this one!